pkg/cmd/infra/router/router.go

```diff
@@ -37,35 +54,75 @@ type templateRouterConfig struct {
 	StatsUsername      string
 }
 
+// f5RouterConfig is the config necessary to start an F5 router plugin
```

Comment:
I get dinged all the time about my godoc not being complete sentences so I'm passing along the favor.  Do a sweep and make sure it is all formatted with //<space> and ends with a period if it is a method or type godoc.